vmdb/spec/models/miq_provision_virt_workflow_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "spec_helper"
+
+describe MiqProvisionVirtWorkflow do
+  context "#continue_request" do
+    let(:sdn) { "SysprepDomainName" }
+    let(:wf)  { FactoryGirl.create(:miq_provision_virt_workflow) }
+
+    before do
+      FactoryGirl.create(:user_admin)
+      MiqDialog.seed
```

Comment:
Creating a workflow requires a dialog and a user.  Also not happy about seeding MiqDialog, but don't know any way around it.  @gmcculloug Any ideas?